__tests__/app.test.js

```diff
@@ -0,0 +1,17 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection");
+
+afterAll(() => db.end());
+
```

Comment:
You will also need to seed the database `beforeEach` test here Thomas - otherwise you'll run into problems when it comes to actually making changes to the database